docassemble/AssemblyLine/al_document.py

```diff
@@ -1,5 +1,5 @@
 import re
-from typing import List, Union
+from typing import Any, Dict, List, Union
 from docassemble.base.util import log, word, DADict, DAList, DAObject, DAFile, DAFileCollection, DAFileList, defined, value, pdf_concatenate, DAOrderedDict, action_button_html, include_docx_template, user_logged_in, user_info, action_argument, send_email, docx_concatenate, reconsider, get_config, space_to_underscore, LatitudeLongitude
```

Comment:
LGTM! But I think we need to import DANav here to fully take advantage of type hints

Reply:
I'm not sure I understand why just including that would help, but I've added it.

Reply:
At least VSCode complains about a type annotation to a type that you haven't included in the .py